.php_cs.dist

```diff
@@ -1,5 +1,15 @@
 <?php
 
+/*
```

Comment:
I don't like the idea of having this header block in a file like `.php_cs.dist`, but oh well...